scripts/azureml-assets/setup.py

```diff
@@ -14,9 +14,7 @@
    install_requires=[
       "GitPython>=3.1",
       "pyyaml>=5",
-      "requests>=2.25",
       "pip>=21",
-      "azure-ai-ml",
    ],
-   python_requires=">=3.6,< 4.0",
+   python_requires=">=3.6,<4.0",

```

Comment:
nit: py>=38 to align with our deprecation strategy?

Reply:
Updated. Thanks!